src/autocomplete_abstract.coffee

```diff
@@ -351,6 +360,10 @@ class AutocompleteAbstract
         $body.append $overlay
         $body.append @floater
 
+        # CSS animation, element only visible when 'animate' is applied
+        if @options.animate
+            setTimeout (=> @floater.addClass('animate') if @floater ), 0
```

Comment:
This seams to have no effect. I can't find the `animate` class you're adding here...